src/locale/properties/signin/signin.en.properties

```diff
@@ -27,7 +27,6 @@ ngOrcid.signin.invalidPassword=Invalid password
 ngOrcid.signin.dontHaveOrcid=Don't have an ORCID iD yet?
 ngOrcid.signin.registerNow=Register now
 ngOrcid.signin.or=or
-ngOrcid.signin.institutionalAccount=Access through your institution
```

Comment:
I think we still use this property.

Reply:
yes, it was duplicated so the script just remove the second one